src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -44,10 +44,17 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
 
   if (asItem) {
     output = (
-      <div title={tooltip} style={style}>
+      <Button
+        type="ghost"
+        style={style}
+        size={24}
+        className={sc('collapsed-control-button')}
```

Comment:
use the existing terminology: `as-item`

or rename asItem to collapsed if you prefer, but we need a single term.